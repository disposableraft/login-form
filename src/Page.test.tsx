import React from "react";
import { render } from "@testing-library/react";
import Page from "./Page";

describe("Page", () => {
  test("renders and matches its snapshot", () => {
    const { container } = render(<Page />);
    expect(container).toMatchSnapshot();
  });
});
