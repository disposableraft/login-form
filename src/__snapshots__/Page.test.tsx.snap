// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders and matches its snapshot 1`] = `
<div>
  <div
    class="css-edqvy"
  >
    <h2
      class="css-84al3w"
      data-testid="welcome"
    >
      Welcome to a login form!
    </h2>
    <div
      class="css-0"
      data-testid="form-login"
    >
      <form
        action="#"
      >
        <div>
          <label
            for="firstName"
          >
            First Name
          </label>
          <input
            id="firstName"
            name="firstName"
            type="text"
            value=""
          />
        </div>
        <div>
          <label
            for="lastName"
          >
            Last Name
          </label>
          <input
            id="lastName"
            name="lastName"
            type="text"
            value=""
          />
        </div>
        <div>
          <label
            for="email"
          >
            Email
          </label>
          <input
            id="email"
            name="email"
            type="email"
            value=""
          />
        </div>
        <div>
          <label
            for="password"
          >
            Password
          </label>
          <input
            id="password"
            name="password"
            type="password"
            value=""
          />
        </div>
      </form>
    </div>
  </div>
</div>
`;
